parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -90,34 +83,6 @@ write_files:
   content: !!binary |
     {{CloudInitData "kubeletSystemdService"}}
 
-- path: /opt/azure/containers/label-nodes.sh
```

Comment:
my concern with making this PR is that we're saying these won't be available for clusters using the Ubuntu distro. In this case for example, making this change means that any clusters using the Ubuntu distro won't be able to deploy k8s 1.16. 1) I'm not sure we want that, 2) if we do we need validation do only allow these features to be used with VHD 3) this might be a problem for existing clusters with these features using the ubuntu distro that won't be able to upgrade after this PR.

Reply:
Good point, we could only exclude all this in a VHD scenario, which we have context in the flow of all these changes.